statistics/update_test.go

```diff
@@ -289,6 +291,21 @@ func (s *testStatsUpdateSuite) TestAutoUpdate(c *C) {
 	c.Assert(stats.Count, Equals, int64(1))
 	c.Assert(stats.ModifyCount, Equals, int64(0))
 
+	_, err = testKit.Exec("insert into t values (1)")
+	c.Assert(err, IsNil)
+	h.DumpStatsDeltaToKV()
+	h.Clear()
+	// We set `Lease` here so that `Update` will use load by need strategy.
+	h.Lease = time.Second
+	h.Update(is)
+	h.Lease = 0
+	err = h.HandleAutoAnalyze(is)
+	c.Assert(err, IsNil)
+	h.Update(is)
+	stats = h.GetTableStats(tableInfo.ID)
```

Comment:
Which assert makes sure that Analyze is not run multiple times?

Reply:
In line 307, we assert that the modify count is 1.

Reply:
Please comment it.